docs/website/gatsby-config.js

```diff
@@ -91,14 +97,24 @@ module.exports = {
         name: 'pages',
       },
     },
-    // {
-    //   resolve: `gatsby-source-filesystem`,
-    //   options: {
-    //     path: `${__dirname}/../nikita/packages`,
-    //     name: 'actions',
-    //     ignore: ['**/assets', '**/env', '**/lib', '**/test']
-    //   },
-    // },
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        path: `${__dirname}/../../packages`,
+        name: 'actions',
+        ignore: [
+          '**/assets',
+          '**/env',
+          '**/lib',
+          '**/test',
+          '**/node_modules',
+          '**/src/utils',
+          '**/core/src/plugins',
+          '**/core/src/schedulers',
+          '**/core/src/session',
+        ]
+      },
+    },
```

Comment:
I am not sure this is the right approach. First, you shall first try to create your own source plugin, second, your globbing expressions are not efficient. No chance for them to optimize the file system discovery.

Reply:
I couldn't find a way to negate ignoring the `core/src/actions` folder. Another way is to call the plugin twice:
```
    {
      resolve: `gatsby-source-filesystem`,
      options: {
        path: `${__dirname}/../../packages`,
        name: 'actions',
        ignore: [
          '**/assets',
          '**/env',
          '**/lib',
          '**/test',
          '**/node_modules',
          '**/src/utils',
          '**/core'
        ]
      },
    },
    {
      resolve: `gatsby-source-filesystem`,
      options: {
        path: `${__dirname}/../../packages/core/src/actions`,
        name: 'actions'
      },
    },
```
I will try with an own plugin